lib/plugins/aws/logs/README.md

```diff
@@ -0,0 +1,42 @@
+# Logs
+
+This plugin returns the CloudWatch logs of a lambda function. You can simply run `serverless logs -f hello` to test it out.
+
+## How it works
+
+`Logs` hooks into the [`logs:logs`](/lib/plugins/logs) lifecycle. It will fetch the CloudWatch log group of the provided function and outputs all the log stream events in the terminal.
+
+## Logs Options
+
+The logs plugin only require the function name you want to fetch the logs for. Other than that there are other options that you can add for extra control:
+
+* **stage:** (shortcut: `-s`) - The stage you want to view the function logs for. If not provided, the plugin will use the default stage listed in `serverless.yaml`. If that doesn't exist either it'll just fetch the logs from the `dev` stage.
+* **region:** (shortcut: `-r`) - The region you want to view the function logs for. If not provided, the plugin will use the default region listed in `serverless.yaml`. If that doesn't exist either it'll just fetch the logs from the `us-east-1` region.
+* **duration:** (shortcut: `-d`) - With this option, you can set a certain duration to fetch the logs. Logs before this time will not be displayed. The default is `30m` (30 minutes). So by default only the logs that happened within 30 minutes will be displayed. You can provide friendly duration strings like `2h` (2 hours), `3d` (3 days) and so on. 
```

Comment:
It's unclear what this param does.

Instead of `--duration` and `--startTime` would be better to have one param. --startTime (BTW why `-m`? :)) that will accept both formats:

- `2h`
- `2010-10-20 4:30`
- other format that are easy to parse